README.md

```diff
@@ -258,6 +258,31 @@ kube-system namespace:
 checkov -d . --skip-check kube-system
 ```
 
+Run a scan of a container image:
+```sh
+checkov --framework sca_image --docker-image sha256:1234example --dockerfile-path /Users/path/to/Dockerfile --bc-api-key ... --repo-id test/test_docker
+```
+OR
+```sh
+checkov --docker-image <image-name>:tag --dockerfile-path /User/path/to/Dockerfile --bc-api-key ...
```

Comment:
the docker-image could also be the hash or the id of the image

Reply:
Worth mentioning that if it's a local docker file, you should build it first, and then extract the hash/id of the image

Reply:
Good idea. Updated.